ddtrace/profiling/collector/_task.pyx

```diff
@@ -0,0 +1,34 @@
+from ddtrace import compat
```

Comment:
I think we could have this module live under `ddtrace/internal`

Reply:
Do you expect != profiler to use it?

Reply:
I could see it but we can always move it later on when that time comes